sloggers/slogtest/t.go

```diff
@@ -36,13 +38,27 @@ type Options struct {
 	// conditions exist when t.Log is called concurrently of a test exiting. Set
 	// to true if you don't need this behavior.
 	SkipCleanup bool
+	// IgnoredErrorValues causes the test logger not to error the test on Error
+	// if the SinkEntry contains one of the listed errors in its "error" Field.
+	// Errors are matched using xerrors.Is().
+	//
+	// By default, context.Canceled and context.DeadlineExceeded are included,
+	// as these are nearly always benign in testing. Override to []error{} (zero
+	// length error slice) to disable the whitelist entirely.
+	IgnoredErrorIs []error
 }
 
 // Make creates a Logger that writes logs to tb in a human readable format.
 func Make(tb testing.TB, opts *Options) slog.Logger {
 	if opts == nil {
 		opts = &Options{}
 	}
+	if opts.IgnoredErrorIs == nil {
+		opts.IgnoredErrorIs = []error{
+			context.Canceled,
+			context.DeadlineExceeded,
+		}
```

Comment:
It might be nice to include this as an exported slice or have some function that produces an `IgnoredErrorIs` slice with these included, so you don't have to manually specify them every time you want to add an extra error instead of overriding them.